runtime/runc_jailer.go

```diff
@@ -156,9 +158,12 @@ func (j *runcJailer) BuildJailedRootHandler(cfg *Config, socketPath *string, vmI
 			newKernelImagePath := filepath.Join(rootPath, kernelImageFileName)
 			j.logger.WithField("newKernelImagePath", newKernelImagePath).Debug("copying kernel image")
 
-			if err := copyFile(m.Cfg.KernelImagePath, newKernelImagePath, 0444); err != nil {
+			if err := copyFile(m.Cfg.KernelImagePath, newKernelImagePath, 0400); err != nil {
 				return errors.Wrap(err, "failed to mount kernel image")
 			}
+			if err := os.Chown(newKernelImagePath, int(j.uid), int(j.gid)); err != nil {
```

Comment:
Do we want to chown it?  I think leaving at 0444 with ownership different to the user inside the jail is probably sufficient.

Reply:
We chown a lot of stuff, including a copied firecracker binary. Would it be better to chown only $OCI_BUNDLE_PATH/rootfs/ where we have firecracker, Linux kernel and drive files?

Reply:
I feel like if we can get 0400 permissions by doing the chown we may as well as it's technically a reduced permission (even if marginally so). I can't see any downside to doing what's in the PR.